CHANGELOG.rst

```diff
@@ -1,10 +1,11 @@
-**3.0.7 - 09/24/24**
+**3.0.7 - 09/25/24**
 
   - Fix mypy errors: vivarium/framework/lifecycle.py
   - Fix mypy errors: vivarium/framework/population/manager.py
   - Fix mypy errors: vivarium/framework/population/population_view.py
   - Enable population manager and population view methods to accept strings
```

Comment:
I should have done this myself, but can you make this the first element in the changelog? It's the only user-facing change, so we should not let it get hidden in the middle.